### PR TITLE
Update Functionality and Styling To Directory

### PR DESCRIPTION
**Pull request by clizzy12**
JIRA Ticket: 
https://scsdean.atlassian.net/browse/WEB2020-43
https://scsdean.atlassian.net/browse/WEB2020-27
https://scsdean.atlassian.net/browse/WEB2020-53

# Description
Updated design to Directory cards so photos will show properly. Also added functionality to have the Directory show in a List/Table view for better searching from the human eye. 

# Test Instructions
1. Go to `/directory`
2. Test out switching between List View and Grid/Card View
3. Make sure Search still works
4. Make sure Lazy Loading Button functionality is correct.

### CODE REVIEW

File: src/views/DirectoryView.vue [hunk 1]
```diff
@@ -69,56 +62,42 @@ export default {
...
       }else{
-        return false
+        return false;
       }
     },
     placeholder() {
       let depTitleHold = this.depTitle ? ' ' + this.depTitle.toLowerCase() : ''
       return 'search ' + this.nav.replace('_', ' ') + depTitleHold + ' names';
+    },
+    finalDirectory(){
```
[the remark this answers is not in the document]
Comment (billyconnerjr) [reply src/views/DirectoryView.vue]: I just checked it again and It appears to be working correctly. I'll see If I can reproduce what I saw earlier.

[the remark this answers is not in the document]
Comment (billyconnerjr) [reply src/views/DirectoryView.vue]: To reproduce the issue I saw.
1. Go to the directory page
2. search for someone , for instance 'martin'
3. This works ok.
4. Now go to the directory page, search for 'zimmerman'.
5. returns nothing
It seems to work the first time you search but then stops.

[the remark this answers is not in the document]
Comment (clizzy12) [reply src/views/DirectoryView.vue]: The real issue was due to a case sensitive search. This has been fixed with the latest commit.

**Comment by clizzy12** [comment]
@billyconnerjr PR had been updated with new Department Filter. Ready for re-review.

**Comment by billyconnerjr** [comment]
Previously tested issues passed

**Pull request merged**